CONTRIBUTING.rst

```diff
@@ -68,10 +65,11 @@ We recommend the following workflow:
    b. Write your test cases and run the complete test suite, see the section
       *Running the test suite* for details.
 
-
    c. Document any user-facing changes in one of the ``/docs/`` files.
+      Please use `one sentence per line`_.
```

Comment:
The URL is named "one sentence per line", but I think that the rule is "one sentence **or** clause per line".

```suggestion
      Please use `semantic linefeeds`_.
```